logic/love_count.py

```diff
@@ -79,12 +79,16 @@ def rebuild_love_count():
     set_toggle_state(LOVE_SENDING_ENABLED, True)
 
 
-def get_love_counts_by_week(username):
+def get_love_counts_by_week(username, unixtime_range_start, unixtime_range_end):
```

Comment:
(nb) Default these to `None` and make the date query (or either end of it) optional